src/utils/breezeUtils.ts

```diff
@@ -0,0 +1,9 @@
+export function isRetriable(statusCode: number): boolean {
+  return (
+    statusCode === 408 || // Timeout
```

Comment:
Isn't partial content `206` retryable as well?

Reply:
:+1: Added to this util. Implemention will follow in next PR